content/project/conduct.adoc

```diff
@@ -98,27 +100,30 @@ Those hosted on Google Groups (see
 link:/mailing-lists[here]) and those self-hosted
 on link:http://lists.jenkins-ci.org/mailman/listinfo[lists.jenkins-ci.org].
 
-==== IRC Channels
+==== Chats
 
-Documented in the
-link:/chat[IRC Channel] wiki
-page, e.g.
+All chats documented on the link:/chat[this page], e.g.
 
-* `#jenkins`
-* `#jenkins-meeting`
-* `#jenkins-infra`
-* `#jenkins-community`
+* IRC channels: `#jenkins`, `#jenkins-meeting`, `#jenkins-infra`, etc.
+* Gitter channels within the link:https://gitter.im/jenkinsci/home[https://gitter.im/jenkinsci/] space
 
 ==== Events
 
-* link:/projects/jam/[Jenkins
-  Area Meetup]
+* link:/projects/jam/[Jenkins Area Meetups]
+* link:/events/online-meetup[Jenkins Online Meetup]
 * Events where Jenkins is officially being represented (e.g.
   link:https://fosdem.org[FOSDEM], link:https://socallinuxexpo.org/[SCaLE])
 * Other Jenkins groups (such as conferences, meetups, and other unofficial
   forums) are encouraged to adopt this Code of Conduct. Those groups must
   provide their own moderators and/or reporting system.
 
+==== Social media
+
+Official Jenkins community accounts and channels, including:
+
+* Twitter: link:https://twitter.com/jenkinsci[@jenkinsci], link:https://twitter.com/jenkins_release[@jenkins_release]
```

Comment:
I don't understand this added portion of the Code of Conduct, is the expectation that these twitter accounts (and their contributors) are to be help to our community expectations of conduct, _or_ is the expectation that anybody interacting with these accounts follow the code of conduct?

Reply:
I agree with @rtyler. I think it needs to be clear that it's conversations with those accounts or something similar.

Reply:
It rather applies to people posting to these account when using their permissions OR when explicitly representing the project (e.g. me telling that "as a Jenkins core maintainer, I think that ...<something bad about the project/community> "). I do not think we can or should set expectations for any interactions, freedom of speech is definitely a part of my open source community definition

Reply:
How is a comment thread in response to a `@jenkinsci` tweet different from a mailing list or chat discussion?